cypress/support/commands.ts

```diff
@@ -119,3 +119,17 @@ Cypress.Commands.add('loadLandingPage', () => {
 
   cy.resetToDefaultLayout();
 });
+
+Cypress.Commands.add('removeWidget', (widgetId: string) => {
+  cy.intercept('PATCH', '**/api/chrome-service/v1/dashboard-templates/*').as(
+    'patchLayout'
+  );
+
+  cy.get(`[data-ouia-component-id="${widgetId}"]`).within(() => {
+    cy.get('[aria-label="widget actions menu toggle"]').click();
```

Comment:
@catastrophe-brandon not blocking - but I wonder if we add a `data` attribute to the menu toggle to eliminate the need to check the aria-label? I'd imagine that text will/can change in the future.